### ISSUE TITLE
Question about binning for depth prediction (DINOv2 + DPT Head)

### ISSUE THREAD

**Issue by smj007** [facebookresearch/dinov2#230]
Hi, 

Thanks for your work, it's truly remarkable. I noticed that while training/running inference using the DPT head, binning is not supported by default. 

https://github.com/facebookresearch/dinov2/blob/6a6261546c3357f2c243a60cfafa6607f84efcb7/dinov2/eval/depth/models/decode_heads/decode_head.py#L157: self.classify is set to False. I do get reasonable results using the same:

<img width="1015" alt="image" src="https://github.com/facebookresearch/dinov2/assets/49593646/1bffb182-48b6-4ca2-ab69-ae53f0e18494">

I want to get a probability distribution for depth over 256 bins. To do this, I set

> self.classify = True
> self.n_bins = 256

Using these settings, all my logits are set to 1, and get a black image for the rendered depth.

<img width="997" alt="image" src="https://github.com/facebookresearch/dinov2/assets/49593646/0ac47118-ad3f-4678-a631-0bb6e324a5c1">

I am using the `render_depth` function (https://github.com/facebookresearch/dinov2/blob/main/notebooks/depth_estimation.ipynb) to visualize the depth maps.

Am I doing something wrong here, or are there any other steps I need to be doing to fix this? Thank you!
